### PR TITLE
services/regulated-assets-approval-server: implement SEP-8 "pending" response

### PR DESCRIPTION
**Pull request by marcelosalloum**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [x] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [x] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [x] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [x] This PR adds tests for the most critical parts of the new functionality or fixes.
* [x] I've updated any docs ([developer docs](https://www.stellar.org/developers/reference/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [x] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [x] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### What

Implement SEP-8 "pending" response. Emails starting with "y" are being marked as "pending" and can't be approved nor rejected for payments above the KYC threshold

### Why

This is the last of the five [SEP-8](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0008.md) responses.

### Known limitations

N/A

### CODE REVIEW

File: services/regulated-assets-approval-server/internal/serve/api_kyc_status_test.go [hunk 1]
```diff
@@ -278,6 +309,42 @@ func TestAPI_GETKYCStatus(t *testing.T) {
 	require.NotNil(t, kycStatusGETResponseApprove.ApprovedAt)
 	require.Nil(t, kycStatusGETResponseApprove.RejectedAt)
 
+	// INSERT new account in db's accounts_kyc_status table; new account was rejected after submiting kyc.
```
Comment (acharb) [inline services/regulated-assets-approval-server/internal/serve/api_kyc_status_test.go:312]: if I'm reading this right:
```suggestion
	// INSERT new account in db's accounts_kyc_status table; new account was pending after submiting kyc.
```

File: services/regulated-assets-approval-server/internal/serve/api_tx_approve_test.go [hunk 2]
```diff
@@ -428,6 +428,38 @@ func TestAPI_txAprove_actionRequiredFlow(t *testing.T) {
...
+	resp = w.Result()
+	assert.Equal(t, http.StatusOK, resp.StatusCode)
+	assert.Equal(t, "application/json; charset=utf-8", resp.Header.Get("Content-Type"))
+
+	body, err = ioutil.ReadAll(resp.Body)
+	require.NoError(t, err)
+	wantBody = `{"result": "no_further_action_required"}`
+	require.JSONEq(t, wantBody, string(body))
+
+	// Step 7: verify transactions with 500+ GOAT are rejected
```
Comment (acharb) [inline services/regulated-assets-approval-server/internal/serve/api_tx_approve_test.go:447]:
```suggestion
	// Step 7: verify transactions with 500+ GOAT are pending
```

File: services/regulated-assets-approval-server/README.md [hunk 3]
```diff
@@ -241,6 +255,16 @@ If their KYC was rejected they should see a rejection response.
 }
 ```
 
+If their KYC was marked as pending they should see a rejection response.
```
Comment (acharb) [inline services/regulated-assets-approval-server/README.md:258]:
```suggestion
If their KYC was marked as pending they should see a pending response.
```

**Review by acharb — APPROVED** [review]
lgtm! 

found a couple comments that need a word change (assuming I'm understanding the code base right, first time looking at it)

File: services/regulated-assets-approval-server/README.md (hunk 3 above)
Comment (marcelosalloum) [reply services/regulated-assets-approval-server/README.md]: https://github.com/stellar/go/pull/3673/commits/1dfa80f03740c97be3edd1fa99eaeae092beb78f

File: services/regulated-assets-approval-server/internal/serve/api_tx_approve_test.go (hunk 2 above)
Comment (marcelosalloum) [reply services/regulated-assets-approval-server/internal/serve/api_tx_approve_test.go]: https://github.com/stellar/go/pull/3673/commits/1dfa80f03740c97be3edd1fa99eaeae092beb78f

File: services/regulated-assets-approval-server/internal/serve/api_kyc_status_test.go (hunk 1 above)
Comment (marcelosalloum) [reply services/regulated-assets-approval-server/internal/serve/api_kyc_status_test.go]: https://github.com/stellar/go/pull/3673/commits/1dfa80f03740c97be3edd1fa99eaeae092beb78f

**Pull request merged**